test/integration/volumes_test.go

```diff
@@ -284,54 +299,56 @@ func TestDetectDiskFormat(t *testing.T) {
 
 	tests := []*struct {
```

Comment:
Isnt `TestDetectDiskFormat` now unecessary because we only use the function exported from `k8s.io/mount-utils` or do you want to make sure that it still detects our LUKS disks correctly?

Reply:
Yes, I kept the tests to ensure that our LUKS disks still work correctly.